reviews/NFTTrader/testing.md

```diff
@@ -0,0 +1,48 @@
+# Testing
+
+
+## Summary
+
+- I could not get these tests to run properly - I'm not sure if there was additional config or project files that were removed for the audit. It would have helped if these had been included.
+
+### Recommendations
+- Looks like the test file starts out with a bunch of helper functions. It might be nice to label this or split them out.
```

Comment:
indeed , almost  all of the project that i have seen in the  Js based applications ( or web3 dapps in general ) use the jest or other frameworks to  structure real TDD   by describing the  initial assumptions of the  entities ( like here is the user wallet address , punk asset to be bought , etc ) and then following sequentially the workflow as its being defined in the docs .

Reply:
agree definitely should have a helper/util file with better documentation to understand what functions are being used/called and in what way. It would serve to really reduce the clutter of the file which is very large and hard to read as a result.